jdi-light/src/main/java/com/epam/jdi/light/elements/base/JdiSettings.java

```diff
@@ -27,8 +27,8 @@
 import static com.epam.jdi.tools.ReflectionUtils.isClass;
 import static com.epam.jdi.tools.ReflectionUtils.isInterface;
 import static java.lang.String.format;
+import static java.util.Collections.singletonList;
```

Comment:
please revert all changes in this file
None of them required for mobile

Reply:
revert